support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYou.jsx

```diff
@@ -227,17 +225,10 @@ const ContributionThankYou = ({
     component: (
       <ContributionThankYouSupportReminder
         email={email}
-        isUsEndOfYearAppeal={isUsEndOfYearAppeal}
       />
     ),
     shouldShow: contributionType === 'ONE_OFF',
   };
-  const SURVEY_END_DATE = new Date(Date.parse('2021-01-31'));
-  const now = new Date();
-  const surveyAction = {
-    component: <ContributionThankYouSurvey />,
-    shouldShow: isUsEndOfYearAppeal && now < SURVEY_END_DATE,
-  };
```

Comment:
I've just deleted this, but if we do want this for US contributors until the 31st I can add it back